### DISCUSSION TITLE
How to cosume cf command line root api?

### DISCUSSION THREAD

**Discussion by daoshihaideba** [cloudfoundry/cloud_controller_ng#3056] [General]
Hi collueages, recently my team and I are searching the possibility of reusing the CF command line and develop it into a UI. And facing some issue.
Here is the questions:
How to get the data result from the root url of cf command line using postman? 
For exmaple, the root url of my cf command line 'cf apps' is ' https://api.cf.eu10.hana.ondemand.com/v3/app'
Is there some way to get the response of the url using postman. If the authentication method if oauth2, where i can get the client id and client secret？ Is there a restful api document relative to it?
Thank you in advance 

<img width="1046" alt="image" src="https://user-images.githubusercontent.com/92013799/200769499-5e66dc23-0cd7-4357-9880-bd2a0ceb9e59.png">

**Comment by philippthun** [comment]
Please have a look at https://v3-apidocs.cloudfoundry.org/version/3.127.0/index.html#authentication

For example, you can get a token with `cf oauth-token` and then send it in an HTTP header to the CF endpoint.
